modules/openCypher-in-gsql/pages/openCypher-in-gsql.adoc

```diff
@@ -324,9 +324,9 @@ WHERE x.friendCount = y.friendCount
 
 * Pattern fragments (x:user) and (y:user) *are not* connected by edge traversal or by sharing vertex variables.
 
-== Key Differences: GSQL vs. OpenCypher
+== Key Differences: GSQL OpenCypher mode vs. OpenCypher
 
-GSQL is strongly typed and **schema-first**, while OpenCypher is **schema-optional**. This fundamental difference impacts how OpenCypher queries behave in each system.
+GSQL OpenCypher is strongly typed and **schema-first**, while OpenCypher is **schema-optional**. This fundamental difference impacts how OpenCypher queries behave in each system.
 
 === Example: OpenCypher Query Behavior
```

Comment:
The example really should have some sample data and the output from GSQL's OpenCypher vs. standard OpenCypher. The verbal description alone may not be that clear.